public/supervision/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
   <script type="module" src="../js/sirius-header.js"></script>
   <link rel="stylesheet" href="../css/all.css">
+  <title>Sirius Header</title>
 </head>
 <body class="govuk-template__body">
   <sirius-header show-finance="false"></sirius-header>
```

Comment:
Would you mind explaining how you're using this 'show-finance'? I don't quite understand why we have a couple of index.html with different hard coded values?